pkg/hooks/hooks.go

```diff
@@ -45,12 +45,13 @@ type PVC struct {
 }
 
 type HookSidecar struct {
-	Image           string           `json:"image,omitempty"`
-	ImagePullPolicy k8sv1.PullPolicy `json:"imagePullPolicy"`
-	Command         []string         `json:"command,omitempty"`
-	Args            []string         `json:"args,omitempty"`
-	ConfigMap       *ConfigMap       `json:"configMap,omitempty"`
-	PVC             *PVC             `json:"pvc,omitempty"`
+	Image           string                           `json:"image,omitempty"`
+	ImagePullPolicy k8sv1.PullPolicy                 `json:"imagePullPolicy"`
+	Command         []string                         `json:"command,omitempty"`
+	Args            []string                         `json:"args,omitempty"`
+	ConfigMap       *ConfigMap                       `json:"configMap,omitempty"`
+	PVC             *PVC                             `json:"pvc,omitempty"`
+	DownwardAPI     v1.NetworkBindingDownwardAPIType `json:"downwardAPI,omitempty"`
```

Comment:
I think that when you add it here, it is exposed to the sidecar-hook annotation [1].

[1] https://github.com/kubevirt/kubevirt/blob/25b37338917f1a7bfbe3fc07d87672ef1e39389c/pkg/hooks/hooks_test.go#L49-L60